BlazorUI/wwwroot/css/open-iconic/README.md

```diff
@@ -0,0 +1,114 @@
+[Open Iconic v1.1.1](http://useiconic.com/open)
+===========
+
+### Open Iconic is the open source sibling of [Iconic](http://useiconic.com). It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](http://useiconic.com/open#icons)
+
+
+
+## What's in Open Iconic?
```

Comment:
Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)